renovate.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "lockFileMaintenance": {
```

Comment:
@JavierSegoviaCordoba what is your recommendation for the configuration here?

Reply:
same arrow meta has